tensorflow/python/grappler/controller.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, item, cluster):
     # order. The order of this collection is deterministic.
     self.important_ops = []
     for name in important_op_names:
+      if isinstance(name, bytes):
```

Comment:
Is this still needed ? item.IdentifyImportantOps() should now return strings and not bytes.

Reply:
I was looking at the latest nightly build and it appeared to be returning bytes.  I haven't built from source to see though.